packages/berry-parsers/sources/syml.ts

```diff
@@ -1,3 +1,6 @@
+// @ts-ignore
+import {load}  from 'js-yaml';
```

Comment:
Even if it may not be particularly important to the security in this case, perhaps the recommended [`safeLoad()`](https://github.com/nodeca/js-yaml#safeload-string---options-) method would be more appropriate given the simpler nature of the Yaml being parsed?

Reply:
Yep, that sounds reasonable 👍